hera_cal/lstbin.py

```diff
@@ -556,6 +621,14 @@ def lst_bin_files(data_files, input_cals=None, dlst=None, verbose=True, ntimes_p
     Nbls_to_load : int, default=None, Number of baselines to load and bin simultaneously. If Nbls exceeds this
         than iterate over an outer loop until all baselines are binned. Default is to load all baselines at once.
     ignore_flags : bool, if True, ignore the flags in the input files, such that all input data in included in binning.
+    average_redundant_baselines : bool, if True, baselines that are redundant between nights will be averaged together.
+                                  When this is set to true, Nbls_to_load is interpreted as the number of redundant groups
+                                  to load simultaneously. The number of data waterfalls can be substantially larger in some
+                                  cases.
+    include_autos : bool, if True, include autocorrelations in redundant baseline averages.
```

Comment:
I'd clarify that autocorrelations are not considered "redundant" with one another.

Reply:
If you do include `average_redundant_baselines` I think the autos do get averaged. For this reason, we want to lstbin autocorrs seperately (at least in this implementation).

Reply:
Oh, ok. Then I'd clarify that that's what's going on in the documentation. 

I was thinking that it is generally useful to have LST-binned autos to get the expected noise level, but if we're already talking about redundant averaging, then maybe that's less useful. So I guess this is fine...

Reply:
It is pretty annoying having separate tomls to lstbin autos so maybe this is a feature we should add.